aspnetcore/mvc/controllers/routing.md

```diff
@@ -257,7 +257,7 @@ REST APIs should use attribute routing to model the app's functionality as a set
 
 Attribute routing uses a set of attributes to map actions directly to route templates. The following `StartUp.Configure` code is typical for a REST API and is used in the next sample:
 
-[!code-csharp[](routing/samples/3.x/main/StartupApi.cs?name=snippet)]
+[!code-csharp[](routing/samples/3.x/main/StartupAPI.cs?name=snippet)]
```

Comment:
I use Linux for most of the work I do here, where case-sensitivity means this didn't resolve.